keyboards/abko/ak84bt/info.json

```diff
@@ -123,8 +123,8 @@
     },
     "usb": {
         "device_version": "0.0.1",
-        "pid": "0x4321",
-        "vid": "0x7654"
+        "pid": "0x0021",
+        "vid": "0x03F6"
```

Comment:
This VID is already assigned.  

Is there a specific reason for wanting to use this?

Reply:
The previous PID and VID were used during the development process and had the values of 0x4321 and 0x7654, respectively. They were not official. We now need to update them to the official PID and VID that we have applied for to avoid any non-compliance issues in the future product usage. Thank you!

Reply:
Where did you apply for it at? 

The reason I ask, is because we use this site for reference as it is very up to date, and accurate:
https://www.the-sz.com/products/usbid/index.php?v=0x03F6

And as you see, it shows 0x03F6 belonging to Olivetti, but also that the company is obsolete/gone. 

The official place to apply is USB-IF, but that's ... expensive ($3-5k, depending).  And that is currently being assigned in the 0x3600 range (or near about).   pid.codes, microchip and other companies sublet PID ranges, but none of these companies are assigned the VID of 0x03F6.

Reply:
Thank you for the information! I will verify the validity and legality of the VID and provide an update if necessary. Thank you!